mesh/accountSettings/urls.py

```diff
@@ -2,5 +2,9 @@
 from . import views as accountsettings_views
 
 urlpatterns = [
-    path("displayTheme", accountsettings_views.display_theme, name="display_theme")
+    path("displayTheme", accountsettings_views.display_theme, name="display_theme"),
+
+    # GET    /api/settings/:account_id
+    # PATCH  /api/settings/:account_id
```

Comment:
Might want to update these comments, as I don't believe this is the correct path begins with api/... for these requests.